scripts/translation/qaxml.e.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
This one is kinda annoying for doc-fr as we have results like:

```

qaxml.e: ./fr/appendices/filters.xml

* &string; -2 +0
* &array; -7 +0
* &boolean; -2 +0
* &integer; -3 +0

  php ./base/scripts/translation/qaxml.e.php --add-ignore=./fr/appendices/filters.xml:5663bc86:3049c997

qaxml.e: ./fr/appendices/ini.core.xml

* &php.version.added; -2 +0

  php ./base/scripts/translation/qaxml.e.php --add-ignore=./fr/appendices/ini.core.xml:55daa03d:917d90e2

qaxml.e: ./fr/appendices/ini.list.xml

* &php.version.added; -1 +0

  php ./base/scripts/translation/qaxml.e.php --add-ignore=./fr/appendices/ini.list.xml:9b82c803:b84704f8

qaxml.e: ./fr/appendices/ini.sections.xml

* &Name; -0 +1
* &Changeable; -0 +1
* &Changelog; -0 +1
```

Which are not really problematic

Reply:
> Not all translations are identical, or use the same conventions. So not all scripts will be of use for all translations. The assumptions of each script are described in each file.

I don't expect this QAs scripts will be forced in all translations. I even don't expect two editors to use the very same conventions inside one manual language. And so, one editor can --add-ignore changes they don't bother or are in files of other maintainers.

These scripts exists to help translations to be more in sync with en, and less prone to breakage. They are **NOT** QA acceptance scripts.